src/main/webapp/app/more-feedback-for-tutor/more-feedback-for-tutor.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
```

Comment:
This component is almost the same as the complaint-for-tutor.component.ts file.
I think it should be possible to use one component for both.

Reply:
Done